Doc/library/tkinter.font.rst

```diff
@@ -47,7 +47,20 @@ The different font weights and slants are:
 
    .. method:: actual(option=None, displayof=None)
 
-      Return the attributes of the font.
+      Returns the actual value for attributes when the given font is used on
+      the widget given by *displayof*. *displayof* accepts any tkinter widget
+      and, if omitted, will default to the main window (in most circumstances,
+      this will be your ``Tk`` instance).
+
+      Due to some differences between operating systems (such as pointsizes
+      and the availability of font families), some values returned by this
+      method may vary from those given as kwargs or similar method (and
```

Comment:
"kwargs" is a jargon. Also "kwargs or similar method" does not look clear. I would write about keyword arguments of the constructor and config().

Why not just copy the manual page with minimal adaptation to Python interface?

Reply:
I rewrote it because using the manual page introduces the licensing issues described by [Tal](https://github.com/python/cpython/pull/22434#issuecomment-700913201). Thank you, I will look at using your phrasing once I have finished reviewing #23781